.github/workflows/ci.yml

```diff
@@ -212,15 +212,18 @@ jobs:
           publish-cmd: |
             if [ "$(pnpm config get registry)" = "https://registry.npmjs.org/" ]; then echo 'Error: Using default registry' && exit 1; else pnpm turbo build $TURBO_ARGS --only && pnpm changeset publish --no-git-tag; fi
 
+      - name: Edit .npmrc [link-workspace-packages=false]
+        run: sed -i -E 's/link-workspace-packages=(deep|true)/link-workspace-packages=false/' .npmrc
+
       - name: Install @clerk/backend in /integration
         if: ${{ steps.task-status.outputs.affected == '1' }}
         working-directory: ./integration
-        run: pnpm init && pnpm add @clerk/backend
+        run: pnpm init && pnpm add @clerk/backend --no-save-workspace
```

Comment:
I couldn't find the `--no-save-workspace` flag in the docs. Can you give a TL;DR of what `link-workspace-packages=false` and this flag do together?

Reply:
Good catch. I don't believe this flag actually does anything now. Removed